src/components/mcp/VoMcpHowTo.vue

```diff
@@ -0,0 +1,55 @@
+<template>
+
+  <p class="mb-4">
+    View how to use this API Key to power your local MCP tools
+    <a class="d-inline-block" href="#" @click.stop.prevent="">
+      here
+      <v-dialog v-slot="{ isActive }" activator="parent" max-width="700">
+        <v-card
+          prepend-icon="mdi-file-document"
+          title="Usage Instructions"
+        >
+          <template #append>
+            <v-icon-btn
+              icon="mdi-close"
+              @click="isActive.value = false"
+            />
+          </template>
+
+          <v-divider />
+
+          <v-card-text>
+            <h3>Use with Claude</h3>
+            TODO:
+          </v-card-text>
+
+          <v-card-text>
+            <h3>Use with Cursor</h3>
+            TODO:
+          </v-card-text>
+
+          <v-card-text>
+            <h3>Use with VSCode</h3>
+            TODO:
+          </v-card-text>
```

Comment:
@johnleider need input for what to put here

Reply:
@AndreyYolkin can I pass this off to you?